redis.conf

```diff
@@ -51,6 +51,7 @@
 #
 # loadmodule /path/to/my_module.so
 # loadmodule /path/to/other_module.so
+# loadmodule /path/to/args_module.so [arg [arg ...]]
```

Comment:
this one can be drop if we don't need it

Reply:
unrelated to this PR, but it's a minor improvement we can keep..